scripts/run_tests.sh

```diff
@@ -32,8 +32,14 @@ function display_result {
   fi
 }
 
-flake8 --max-complexity 10 --count
```

Comment:
this was also doing a McCabe complexity analysis, does pylint have the same plugin?